lib/core/game.rb

```diff
@@ -1,32 +1,31 @@
 require "core/board"
-require "core/player"
 
 class Game
-  attr_accessor :board, :currentPlayer
+  attr_accessor :board, :current_player
 
   def initialize(player1, player2)
     @player1 = player1
     @player2 = player2
-    @currentPlayer = player1
-    @board = Board.new(3) # ATM, hardcoded. Down the road, we inyect it.
+    @current_player = player1
+    @board = Board.new(3) # ATM, hardcoded. Down the road, we inject it.
```

Comment:
Here's this 3 again. Seems like you've abstracted away from the hard-coded 3 in at least one place, but not the rest. I'd suggest being consistent: none or all. And that comment reads like an excuse for not doing that work now.